sfyra/pkg/tests/tests.go

```diff
@@ -96,7 +96,7 @@ func Run(ctx context.Context, cluster talos.Cluster, vmSet *vm.Set, capiManager
 		},
 		{
 			"TestCompatibilityCluster",
-			TestCompatibilityCluster(ctx, metalClient, cluster, vmSet, capiManager, options.PrevTalosRelease, options.KubernetesVersion),
+			TestCompatibilityCluster(ctx, metalClient, cluster, vmSet, capiManager, options.PrevTalosRelease, "v1.22.12"),
```

Comment:
shouldn't the k8s version come from some config?

Reply:
Yes, it does, but this test specifically boots old Talos, so we need old Kubernetes

Reply:
so the long story was that previously we used same version for "old" Talos (0.13) and "new" Talos (0.14+). As we're right now on Talos 1.1, which is Kubernetes 1.24, it's no longer compatible with 0.13, so I have to pin old version. Hopefully we should be able to remove this test sooner or later, as it's only needed for Talos 0.13 which doesn't have SideroLink (so it's based on init nodes).